src/ImageSharp/Formats/Jpeg/PdfJsPort/Components/ScanDecoder.cs

```diff
@@ -0,0 +1,887 @@
+                ref short fastACRef = ref MemoryMarshal.GetReference(fastACTables.Tables.GetRowSpan(component.ACHuffmanTableId));
+
+                int mcu = 0;
+                for (int j = 0; j < h; j++)
+                {
+                    for (int i = 0; i < w; i++)
+                    {
+                        if (this.eof)
+                        {
+                            return;
+                        }
+
+                        int blockRow = mcu / w;
+                        int blockCol = mcu % w;
+                        int offset = component.GetBlockBufferOffset(blockRow, blockCol);
+
+                        if (this.spectralStart == 0)
+                        {
+                            this.DecodeBlockProgressiveDC(component, ref Unsafe.Add(ref blockDataRef, offset), ref dcHuffmanTable);
+                        }
+                        else
+                        {
+                            this.DecodeBlockProgressiveAC(ref Unsafe.Add(ref blockDataRef, offset), ref acHuffmanTable, ref fastACRef);
+                        }
+
+                        mcu++;
+
+                        // Every data block is an MCU, so countdown the restart interval
+                        if (--this.todo <= 0)
+                        {
+                            if (this.codeBits < 24)
+                            {
+                                this.GrowBufferUnsafe();
+                            }
+
+                            // If it's NOT a restart, then just bail, so we get corrupt data
+                            // rather than no data
+                            if (!this.ContinueOnRestart())
+                            {
+                                return;
+                            }
+
+                            this.Reset();
+                        }
+                    }
+                }
+            }
+            else
+            {
+                // Interleaved
+                int mcu = 0;
+                int mcusPerColumn = frame.McusPerColumn;
+                int mcusPerLine = frame.McusPerLine;
+                for (int j = 0; j < mcusPerColumn; j++)
+                {
+                    for (int i = 0; i < mcusPerLine; i++)
+                    {
+                        // Scan an interleaved mcu... process components in order
+                        for (int k = 0; k < this.componentsLength; k++)
+                        {
+                            PdfJsFrameComponent component = this.components[k];
+                            ref short blockDataRef = ref MemoryMarshal.GetReference(MemoryMarshal.Cast<Block8x8, short>(component.SpectralBlocks.Span));
+                            ref PdfJsHuffmanTable dcHuffmanTable = ref dcHuffmanTables[component.DCHuffmanTableId];
+                            ref PdfJsHuffmanTable acHuffmanTable = ref acHuffmanTables[component.ACHuffmanTableId];
```

Comment:
`acHuffmanTable` and `fastAC`  are unused

Reply:
Sometimes I miss Resharper.

Reply:
Got the unused, different line though so Github won't see it.